### PR TITLE
Interactive DST exploring

### PR DESCRIPTION
**Pull request by betatim**

### CODE REVIEW

File: code/dst-explore.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# SetupProject DaVinci v36r6
```
Comment (alexpearce) [inline code/dst-explore.py:1]: I guess you know of `dst-explorer.py` script in Bender; is this script better?

Comment (betatim) [inline code/dst-explore.py:1]: I know of `dst-explorer.py` but it was too difficult/intransparent to use
with its lack of docs hence I started my own which just starts python,
opens the DST and then leaves you to do what you want to do. The Bender
script seems to do a lot of magic, takes/took "for ever" to start, had no
docs, ...

File: code/dst-explore.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        nodenames.append(node.identifier())
+
+    return nodenames
+
+def advance(decision='B02DKWSD2HHHBeauty2CharmLine'):
+    """Advance until stripping decision is true, returns
+    number of events by which we advanced"""
+    n = 0
+    while True:
+        appMgr.run(1)
```
Comment (apuignav) [inline code/dst-explore.py:46]:
``` python
if not evt["/Event/DAQ/RawEvent"]:
    print "Reached end of input files"
    break
```

**Comment by betatim** [comment]
I haven't added the rendered HTML yet, will do as soon as we are happy (makes the diffs easier to read).

From my side it is ready to merge, otherwise it will become huge.

**Comment by apuignav** [comment]
Merged. Let's work on this now.

**Comment by betatim** [comment]
I added the rendered HTML directly to `gh-pages`.

**Comment by alexpearce** [comment]
I was going to say: change “python” to “Python”.

**Pull request merged**